### PR TITLE
Boomee🌸- Addition

### PR DESCRIPTION
**Pull request by HemantKArya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -522,6 +522,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 ### • Music & Audio Player
 
 * [**Auxio**](https://github.com/OxygenCobalt/Auxio) <sup>**[[F-Droid](https://f-droid.org/app/org.oxycblt.auxio)]**</sup>
+* [**Bloomee🌸**](https://github.com/HemantKArya/BloomeeTunes) <sup>**[[SourceForge](https://bloomee.sourceforge.io)]**</sup>
```
Comment (offa) [inline README.md:525]: (Until now) please don't use emojis to stay consistent with the other entries. Also , the `sup` part is for F-Droid / Izzy only.

You should consider a submission btw. :smiley:.

**Review by offa — CHANGES_REQUESTED** [review]
Thanks for the submission! There are some minor editorial things, but otherwise ok :+1:

File: README.md (hunk 1 above)
Comment (offa) [reply README.md:525]: You don't have to :-). Remove Emoji and the `<sup>…</sup>` and this PR is ready to merge.

F-Droid / Izzy is fully optional.

Comment (HemantKArya) [reply README.md]: Okay that sounds good too.

**Review by offa — APPROVED** [review]
Perfect :+1:

**Pull request merged**